libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.spec.ts

```diff
@@ -72,28 +71,26 @@ describe('Toggle switch component', () => {
       ).toEqual(false);
     });
 
-    it('should toggle `checked` state on click', fakeAsync(() => {
```

Comment:
There were lots of unnecessary `fakeAsync`/`tick` calls in these tests that I removed. The `ngModel` and reactive form tests still required them, so I left those.